src/list.rs

```diff
@@ -37,6 +38,10 @@ impl Entry {
     pub fn simple_type(&self) -> Option<SimpleType> {
         self.file_type
     }
+    /// Returns the inode number of this entry
+    pub fn inode(&self) -> u64 {
```

Comment:
Should be `libc::ino_t` I think.

Reply:
I deliberately chosen the types from statx() (see #38) these are overall better suited than
the legacy types (which is a pain to deal with when differing in signedness). Arguably it
would been nice if statx() would define sx_ino_t or similar, but they are only aliases so
there isn't overly much value in doing so. In anticipation of statx() it would be a bit
awkward to return types as in struct stat which are sometimes inferior and kindof legacy.

Reply:
Note `statx` is a linux-specific call. This library also works on macos and different bsd variants.

Other than that, using libc types is convenient if you want to use other libc functions on that types. Because otherwise it will work on some platform but have compilation errors on others.

Reply:
I am aware that statx() is linux only. but the types it uses are well chosen to be a superset of what 'struct stat' provides. My plan was to provide 'accessor' functions that use these statx() types even when statx() isn't available.

 I have some hopes that eventually other vendors pick this up and it becomes standardized. Anyway it could be changed to 'struct stat' compatibility. (also in #38)

Reply:
Pushed a new version which uses ino_t, removed the origin experiment.
The later will need more work and a complete different approach.